### PR TITLE
Add opticon back with corrected nav styling

### PR DESCRIPTION
**Pull request by stewsmith**
This code adds back what was reverted last night plus the white nav correction.

### CODE REVIEW

File: website-guts/assets/css/modules/opticon-header.scss [hunk 1]
```diff
@@ -0,0 +1,49 @@
+#opticon-nav {
+  background-color: white;
```
Comment (stewsmith) [inline website-guts/assets/css/modules/opticon-header.scss]: This is the change in specificity.

File: website-guts/assets/css/modules/opticon-header.scss [hunk 2]
```diff
@@ -0,0 +1,49 @@
...
+      a {
+        display: block;
+        text-decoration: none;
+      }
+    }
+  }
+}
+
+@media only screen and (min-width: $tablet-min-width) {
+  #opticon-nav {
```
Comment (stewsmith) [inline website-guts/assets/css/modules/opticon-header.scss]: This is the change in specificity.

File: website-guts/templates/layouts/opticon.hbs [hunk 3]
```diff
@@ -0,0 +1,19 @@
+---
+layout: wrapper.hbs
+layout_script: false
+layout_body_class: false
+---
+
+<nav id="opticon-nav">
```
Comment (stewsmith) [inline website-guts/templates/layouts/opticon.hbs]: This is the change in specificity.

**Comment by dtothefp** [comment]
`opticon/speakers/` grid layout is breaking in mobile/tablet and text not centering. Also, why is that form for the dropdown `select` making the whole page flicker?

screen shot 2015-01-14 at 1 20 49 pm

screen shot 2015-01-14 at 1 20 39 pm

**Comment by stewsmith** [comment]
@dtothefp The grid displays properly on iPad and iPad mini.

I'm not sure about the flickering you're talking about. Could you attach a video?

**Comment by dtothefp** [comment]
@stewsmith it looks as though you have width's hardcoded in px for tablet and mobile width.  The fact it works in IPad is interesting but not a flexible solution, you could probably do this better with percentages. Also, text align should be centered in tablet and mobile

screen shot 2015-01-14 at 1 46 23 pm
 
Text areas of forms should probably have their widths match inputs in tablet 

screen shot 2015-01-14 at 1 44 54 pm

**Comment by dtothefp** [comment]
@stewsmith flashing form fields vid http://cl.ly/image/0x0r402e2R3g

**Pull request merged**